hazelcast/src/test/java/com/hazelcast/config/YamlOnlyConfigBuilderTest.java

```diff
@@ -111,6 +112,7 @@ public void testExplicitNullScalarThrows() {
     }
 
     @Test
+    @Ignore
```

Comment:
Actually, no. This tests the case when the config is not under a root `hazelcast` root object, but we won't keep supporting this behavior (see the Breaking changes in the PR description).
I removed the test now instead of just ignoring it.

Reply:
I see. If it's impossible with the schema, I would opt to duplicate the test - in a separated serial test class - for the with and without schema cases. The code handling the omitted root node is still there and I guess we support it with the schema check disabled.